drivers/backup/portworx/portworx.go

```diff
@@ -1254,6 +1254,32 @@ var (
 			"container":         {"", ""},
 		},
 		},
+		"mysql-backup": {"pre": {"pre_action_list": {"mysql --user=root --password=$MYSQL_ROOT_PASSWORD -Bse 'FLUSH TABLES WITH READ LOCK;system ${WAIT_CMD};'"},
+			"background":        {"false"},
```

Comment:
background is true for pre-rule as per docs.

Reply:
corrected